### PR TITLE
Raz/mgmt 1417-ResettingPendingUserAction

### PR DESCRIPTION
**Pull request by RazRegev**
http://10.0.9.195:8080/job/bm-inventory-subsystem-test/466/console

### CODE REVIEW

File: internal/host/host.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ const (
 	HostStatusInstalling                  = "installing"
 	HostStatusInstallingInProgress        = "installing-in-progress"
 	HostStatusInstallingPendingUserAction = "installing-pending-user-action"
+	HostStatusResettingPendingUserAction  = "resetting-pending-user-action"
```
Comment (filanov) [inline internal/host/host.go:45]: suggesting to stop adding consts to this list and start using `models.HostStatus...`

File: internal/host/host.go [hunk 2]
```diff
@@ -355,6 +370,16 @@ func (m *Manager) CancelInstallation(ctx context.Context, h *models.Host, reason
 	return nil
 }
 
+func (m *Manager) IsRequireManualReset(h *models.Host) bool {
+	if *h.Status != HostStatusResetting {
```
Comment (filanov) [inline internal/host/host.go:374]:
```suggestion
	if *h.Status != models.HostStatusResetting {
```

File: internal/host/host.go [hunk 3]
```diff
@@ -367,6 +391,17 @@ func (m *Manager) ResetHost(ctx context.Context, h *models.Host, reason string,
 	return nil
 }
 
+func (m *Manager) ResetHostManually(ctx context.Context, h *models.Host, db *gorm.DB) error {
```
Comment (eranco74) [inline internal/host/host.go:394]: Name is misleading.
Consider renaming to "ResetPendingUserAction"

File: internal/cluster/insufficient_test.go [hunk 4]
```diff
@@ -53,6 +53,19 @@ var _ = Describe("insufficient_state", func() {
...
+			addHost(models.HostRoleMaster, models.HostStatusResetting, *cluster.ID, db)
+			c := geCluster(*cluster.ID, db)
+			Expect(len(c.Hosts)).Should(Equal(1))
+			updateHostProgress(c.Hosts[0], models.HostStageRebooting, "rebooting", db)
+			updateReply, updateErr = state.RefreshStatus(ctx, &cluster, db)
+			Expect(updateErr).Should(BeNil())
+			Expect(updateReply.State).Should(Equal(clusterStatusInsufficient))
+			c = geCluster(*cluster.ID, db)
+			Expect(c.Hosts[0].Progress.CurrentStage).Should(Equal(models.HostStageRebooting))
+			Expect(swag.StringValue(c.Hosts[0].Status)).Should(Equal(models.HostStatusResettingPendingUserAction))
```
Comment (eranco74) [inline internal/cluster/insufficient_test.go:66]: Where is the code that handle the transition from HostStatusResettingPendingUserAction?
Add a test for that transition as well

**Pull request merged**